src/Actions/AuthActions.js

```diff
@@ -1,27 +1,25 @@
-import axios from "axios";
-import { Login, LoginFailed } from "../ActionCreators/AuthActionCreators";
+import axios from 'axios';
+import { Login, LoginFailed } from '../ActionCreators/AuthActionCreators';
 
-const BASE_URL = "https://api.openconceptlab.org/";
+const BASE_URL = 'https://api.openconceptlab.org/';
 
 // login action
-const LoginAction = ({ username, token }) => {
-  return async dispatch => {
-    try {
-      const url = `${BASE_URL}/users/${username}/`,
-        headers = { Authorization: `Token ${token}` };
-      // dispatch login sent
-      const response = await axios.post(url, null, { headers });
-      dispatch(Login(response));
-      localStorage.setItem("token", `Token ${token}`);
-      localStorage.setItem("username", `${username}`);
-    } catch (error) {
-      if (error.response) {
-        dispatch(LoginFailed(error.response.data.detail));
-      } else if (error.request) {
-        dispatch(LoginFailed("Request can't be made"));
-      }
+const LoginAction = ({ username, token }) => async (dispatch) => {
```

Comment:
Correct me if I am wrong but I think function/method names should be written camelCase because I see TableCase almost everywhere. That's if you are following the airbnb style guide though

Reply:
Thank you for the feedback. Let me work on the correcting the naming.

Reply:
I finished correcting the naming of the methods following the conventions